### PR TITLE
Add advisory speed limit data and pics

### PR DESCRIPTION
**Pull request by rugk**
One step for https://github.com/westnordost/StreetComplete/issues/470

I've left the data still in a draft way and include comments and so one… (Making it hopefully a bit easier to review.)
We may e.g. remove the "unknown below" as they are implicitly shown by the rest of the file.

However, I would leave (other) the `false` entries in the `isAdvisorySpeedLimitKnown.yml` even though the default is set to false as they show that for this country data was available and it is deliberately "false" and does not need research, i.e. no data is missing.
Of course, the entries can also be sorted.

### CODE REVIEW

**Comment by rugk** [comment]
Ah right, so most are SVGs so this should be possible without big problems. (and then ignore the Chinese version and alos use the yellow one, as I commented it is used there too)

Maybe you can still merge the PR into a new branch or so, so you do not have to leave it around here? (You – and I – should be able to change the branch at the top.)

File: res/authors.txt [hunk 1]
```diff
@@ -18,6 +18,16 @@ building_levels_illustration.svg
...
 
+advisory_speed_limit_blue.svg	Andreas 06, Mediatus; CC0L; https://en.wikipedia.org/wiki/File:Zeichen_380-52_-_Richtgeschwindigkeit_80_km-h,_StVO_1992.svg
+
+advisory_speed_limit_yellow.svg	Standards Australia; Public Domain; 
+
+advisory_speed_limit_white.svg	Public Domain; https://en.wikipedia.org/wiki/File:Philippines_road_sign_W8-1.svg
+
+advisory_speed_limit_white_horizontal.svg	UK Traffic signs image database; Open Government Licence v1.0; https://en.wikipedia.org/wiki/File:UK_traffic_sign_513.2.svg
+
+advisory_speed_limit_china		Beijing Traffic Management Bureau, https://commons.wikimedia.org/wiki/File:China_road_sign_%E8%B7%AF_62a.jpg
```
Comment (rugk) [inline res/authors.txt]: You deleted the file. So you can also remove this line.

File: res/authors.txt [hunk 2]
```diff
@@ -18,6 +18,16 @@ building_levels_illustration.svg
 
 living_street.svg
 
+advisory_speed_limit_blue.svg	Andreas 06, Mediatus; CC0L; https://en.wikipedia.org/wiki/File:Zeichen_380-52_-_Richtgeschwindigkeit_80_km-h,_StVO_1992.svg
+
+advisory_speed_limit_yellow.svg	Standards Australia; Public Domain; 
+
+advisory_speed_limit_white.svg	Public Domain; https://en.wikipedia.org/wiki/File:Philippines_road_sign_W8-1.svg
+
+advisory_speed_limit_white_horizontal.svg	UK Traffic signs image database; Open Government Licence v1.0; https://en.wikipedia.org/wiki/File:UK_traffic_sign_513.2.svg
```
Comment (rugk) [inline res/authors.txt]: You deleted the file. So you can also remove this line.

File: res/authors.txt [hunk 3]
```diff
@@ -18,6 +18,16 @@ building_levels_illustration.svg
 
 living_street.svg
 
+advisory_speed_limit_blue.svg	Andreas 06, Mediatus; CC0L; https://en.wikipedia.org/wiki/File:Zeichen_380-52_-_Richtgeschwindigkeit_80_km-h,_StVO_1992.svg
+
+advisory_speed_limit_yellow.svg	Standards Australia; Public Domain; 
+
+advisory_speed_limit_white.svg	Public Domain; https://en.wikipedia.org/wiki/File:Philippines_road_sign_W8-1.svg
```
Comment (rugk) [inline res/authors.txt]: You renamed this file.

Comment (westnordost) [inline res/authors.txt]: Right

Comment (rugk) [inline res/authors.txt]: (see also other comments…)

**Pull request merged**